src/components/com_tjcertificate/site/tjcertificate.php

```diff
@@ -15,6 +15,7 @@
 use Joomla\CMS\MVC\Controller\BaseController;
 
 JLoader::import('components.com_tjcertificate.includes.tjcertificate', JPATH_ADMINISTRATOR);
+TJCERT::init();
```

Comment:
add this in the admin side entry file as well

Reply:
done